R/plot.R

```diff
@@ -1135,14 +1135,12 @@ plot.netdx <- function(x, type = "formation", method = "l", sims, stats,
 
     ## Joined Plots
     if (method == "l") {
-
-      if (missing(sim.lines)) {
-        if (dynamic == TRUE) {
-          sim.lines <- FALSE
-        } else {
-          sim.lines <- TRUE
-        }
+      if(missing(sim.lines)){
+        sim.lines <- FALSE
       }
+       else {
+          simlines <- TRUE
+       }
```

Comment:
Note the variable name error

Reply:
I'm not sure if this was the best solution, but I put dynamic=TRUE conditions when quantile calculations are being done; would it make more sense/cleaner to simply separate the two conditions from each other (dynamic = TRUE vs dynamic = FALSE).